src/decorator/relations/ManyToOne.ts

```diff
@@ -2,31 +2,31 @@ import {getMetadataArgsStorage, ObjectType, RelationOptions} from "../../";
 import {RelationMetadataArgs} from "../../metadata-args/RelationMetadataArgs";
 
 /**
- * Many-to-one relation allows to create type of relation when Entity1 can have single instance of Entity2, but
- * Entity2 can have a multiple instances of Entity1. Entity1 is an owner of the relationship, and storages Entity2 id
- * on its own side.
+ * A many-to-one relation allows creating the type of relation where Entity1 can have a single instance of Entity2, but
+ * Entity2 can have multiple instances of Entity1. Entity1 is the owner of the relationship, and stores the id of
+ * Entity2 on its side of the relation.
  */
 export function ManyToOne<T>(typeFunctionOrTarget: string|((type?: any) => ObjectType<T>), options?: RelationOptions): PropertyDecorator;
 
 /**
- * Many-to-one relation allows to create type of relation when Entity1 can have single instance of Entity2, but
- * Entity2 can have a multiple instances of Entity1. Entity1 is an owner of the relationship, and storages Entity2 id
- * on its own side.
+ * A many-to-one relation allows creating the type of relation where Entity1 can have a single instance of Entity2, but
+ * Entity2 can have multiple instances of Entity1. Entity1 is the owner of the relationship, and stores the id of
+ * Entity2 on its side of the relation.
  */
 export function ManyToOne<T>(typeFunctionOrTarget: string|((type?: any) => ObjectType<T>),
                              inverseSide?: string|((object: T) => any),
                              options?: RelationOptions): PropertyDecorator;
 
 /**
- * Many-to-one relation allows to create type of relation when Entity1 can have single instance of Entity2, but
- * Entity2 can have a multiple instances of Entity1. Entity1 is an owner of the relationship, and storages Entity2 id
- * on its own side.
+ * A many-to-one relation allows creating the type of relation where Entity1 can have a single instance of Entity2, but
+ * Entity2 can have multiple instances of Entity1. Entity1 is the owner of the relationship, and stores the id of
+ * Entity2 on its side of the relation.
  */
 export function ManyToOne<T>(typeFunctionOrTarget: string|((type?: any) => ObjectType<T>),
                              inverseSideOrOptions?: string|((object: T) => any)|RelationOptions,
                              options?: RelationOptions): PropertyDecorator {
 
-    // normalize parameters
+    // Normalize parameters.
```

Comment:
there is no need to upper-case inline comment and put a dot to the end. This is a default code style in all TypeORM source files.